styles/layout.css

```diff
@@ -0,0 +1,121 @@
+@media(max-width: 768px) {
+    .a{
+        width: 480px;
+        height: 75px;
+        background-color: red;
+    }
+    .b{
+        width: 480px;
+        height: 150px;
+        background-color: blue;
+        
+    }
+    .top{
```

Comment:
i would probably only have things like the top and center class in layout.css .. your a, b, c etc are actually closer to modules than layout.